backend/src/migrations/20230828154649_update_user_details_table.ts

```diff
@@ -0,0 +1,26 @@
+import { type Knex } from 'knex';
+
+import { CurrencyType } from '~/common/enums/enums.js';
```

Comment:
For migration don't use imported enums because someone will change this in future and we will have wrong migration